src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -251,6 +230,7 @@ static Task StopEndpoints(IEnumerable<ComponentRunner> endpoints)
             var stopTimeout = TimeSpan.FromMinutes(2);
             return endpoints.Select(async endpoint =>
             {
+                await Task.Yield(); // ensure all endpoints are stopped even if a synchronous implementation throws
```

Comment:
@danielmarbach thoughts on this? I think it's necessary to prevent the select loop exiting early because of a synchronous implementation of `Stop()` throwing an exception?